calico-policies/private-network-isolation/eu-es/allow-ibm-ports-private.yaml

```diff
@@ -0,0 +1,88 @@
+# This policy opens port 10250 for VPN communication between master and workers,
+# port 53 for DNS (5353 for OpenShift 4.3 and later), port 52311 for IBM BigFix
+# worker node updates, port 2049 forcommunication with NFS file servers,
+# port 3260 for communication to block storage, port 5473 for communication to
+# the calico-typha ClusterIP, port 2040 and 2041 on 172.20.0.0/24 for the master
+# API server local proxy, and port 443 on 172.21.0.1 (or 10.10.10.1 for clusters
+# created more than 3 years ago) for the etcd local proxy.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-ibm-ports-private
+spec:
+  egress:
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 2049
+      - 3260
+      - 5473
+    protocol: UDP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 2049
+      - 3260
+      - 5473
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: UDP
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: TCP
+  - action: Allow
+    destination:
+      ports:
+      - 443
+      nets:
+      - 172.21.0.1/32 # For clusters created <3 years ago
+      - 10.10.10.1/32 # For clusters created >3 years ago
```

Comment:
Since this is a new region, we won't have any clusters >3 years old, so please remove this (make it look like the br-sao verison of this file)